src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int legA;
+    private int legB;
+
+    public RightTriangle(int legA, int legB, String color) {

```

Comment:
Consider using the `Color` enum instead of a `String` for the color parameter to maintain type safety and consistency with the rest of the code.